appservice/src/main/java/nu/yona/server/subscriptions/rest/UserController.java

```diff
@@ -272,6 +280,12 @@ static ControllerLinkBuilder getAddUserLinkBuilder()
 		return linkTo(methodOn.addUser(Optional.empty(), null, null, null));
 	}
 
+	static ControllerLinkBuilder getUpdateUserLinkBuilder(UUID userId)
+	{
+		UserController methodOn = methodOn(UserController.class);
+		return linkTo(methodOn.updateUser(Optional.empty(), null, userId, null, null));
```

Comment:
Yes, it does make sense that the single user has a low frequency which can be asserted over a larger window. Even a window that lasts for a day can be used to restrict the frequency even further.

For new users, one IP might indeed register multiple users if a proxy is used. I hope that in the Yona promotion meetings they have different IP addresses though... The advantage of having a shorter window is that the server does not have to remember `addUser` calls by IP address that long, so can free memory. However, because `addUser` calls do not occur many times, it is also not very much of a problem to have a large window here too. So maybe we could still use the same large window (e.g. 1 hour or 1 day) for both cases.

The advantage of having the same window is that there are less settings to think about. The window setting only influences the memory on the server, not the frequency actually. Only if there are very infrequent calls versus very frequent calls it makes sense to use 2 windows, a shorter window for the very frequently called method and a longer window for the very infrequently called method. Do you think the `addUser` is called much more frequently than `updateUser`?